src/session.rs

```diff
@@ -14,11 +14,14 @@ use crate::{
     requests::request::PreparedStatementWrapper, result::QueryResultWrapper, utils::err_to_napi,
 };
 
+const DEFAULT_CACHE_SIZE: u32 = 500;
```

Comment:
:pick: For better alignment, let's use multiples of 2.

```suggestion
const DEFAULT_CACHE_SIZE: u32 = 512;
```